tracer/src/Datadog.Trace.BenchmarkDotNet/DatadogExporter.cs

```diff
@@ -110,6 +111,30 @@ public IEnumerable<string> ExportToFiles(Summary summary, ILogger consoleLogger)
                         testName += report.BenchmarkCase.Parameters.DisplayInfo;
                     }
 
+                    // The Job Id can contain random values: https://github.com/dotnet/BenchmarkDotNet/blob/ec429af22e3c03aedb4c1b813287ef330aed50a2/src/BenchmarkDotNet/Jobs/JobIdGenerator.cs#L18
+                    // Example:
+                    //      Job-WISXTD(Runtime=.NET Framework 4.7.2, Toolchain=net472, IterationTime=2.0000 s)
+                    // We use the description as part of the configuration facets.
+                    // The configuration facets are used by CI Visibility to create a fingerprint of the test.
+                    // Random values here means a new test fingerprint every time, so there's no way to track the same
+                    // test in multiple executions. So because of that, we need to remove the random part of the description.
+                    // In this case we remove the job name and keep the information between the parethesis.
+                    var description = report.BenchmarkCase.Job?.DisplayInfo;
+                    if (description is not null)
+                    {
+                        var match = new Regex(@".*\((.*)\)").Match(description);
```

Comment:
This regex is maybe a bit too lenient? it works as long as there's only one pair of brackets, but if the description is updated to e.g. 

```
Job-WISXTD(Runtime=.NET Framework 4.7.2, Toolchain=net472) Parameters(IterationTime=2.0000 s, Iterations=56)
```

(a random potential example), then we would only use the final bracket, and the description would be:

```
IterationTime=2.0000 s, Iterations=56
``` 

Maybe we should use something like this instead (lazy match), so that we get the contents of both brackets?

```suggestion
                        var match = new Regex(@".*?\((.*)\)").Match(description);
```

Which would give the same result _currently_, but for the update above, would give 

```
Runtime=.NET Framework 4.7.2, Toolchain=net472) Parameters(IterationTime=2.0000 s, Iterations=56
```

(A bit ugly with the middle brackets, but includes all the info). Alternatively, you could use `Matches` and go for something like this:

```suggestion
                        var match = new Regex(@"\((.*?)\)").Matches(description);
                        if (match.Count > 0)
                        {
                            description = string.Join(", ", match.Select(x => x.Groups[1].Value));
                        }
```

That would give the same result for your initial example, and for my contrived example it would be 

```
Runtime=.NET Framework 4.7.2, Toolchain=net472, IterationTime=2.0000 s, Iterations=56
```

which is probably the nicest. But you know, I went to far down the Regex rabbit hole here, so feel free to ignore me 😅

Reply:
Ok I changed and fix it to support this case in https://github.com/DataDog/dd-trace-dotnet/pull/3960/commits/6817366aa2745c3d6f18e9f943f221f33ebdfc34

I tested the regex here: https://dotnetfiddle.net/Ln9tVn

Reply:
👍 LGTM